### PR TITLE
Refactor date formatting

### PR DESCRIPTION
**Pull request by SergioEstevao**
This PR just does some refactor around date/time formatting code to avoid code duplication and make things more consistent for our users.

To test:
 - Run the demo app
 - See if the time on videos shows correctly on thumbs cells and when previewing video full screen.

### CODE REVIEW

**Comment by frosty** [comment]
PR looks great, nice cleanup of the code!

While testing, I found a small issue with the time formatting, which perhaps we could clean up at the same time?

img_7067

**Comment by SergioEstevao** [comment]
@frosty ready for another look.

File: Example/Tests/WPDateTimeHelpersTests.m [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#import <XCTest/XCTest.h>
+#import "WPDateTimeHelpers.h"
+
+@interface NSString ()
++ (NSString *)emojiCharacterFromCoreEmojiFilename:(NSString *)filename;
```
Comment (SergioEstevao) [inline Example/Tests/WPDateTimeHelpersTests.m]: Bad copy paste...

**Comment by SergioEstevao** [comment]
I will go for the formatter approach looks more consistent.

**Comment by SergioEstevao** [comment]
@frosty ready for another look.

**Pull request merged**